vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageImageVideoItem.kt

```diff
@@ -66,7 +66,23 @@ abstract class MessageImageVideoItem : AbsMessageItem<MessageImageVideoItem.Hold
         } else {
             RoundedCorners(dimensionConverter.dpToPx(8))
         }
-        imageContentRenderer.render(mediaData, mode, holder.imageView, imageCornerTransformation)
+
+        val isImageMessage = attributes.informationData.messageType == MessageType.MSGTYPE_IMAGE
+        val autoplayAnimatedImages = attributes.autoplayAnimatedImages
+
+        var resolvedMode = mode
+
+        holder.playContentView.visibility = if (playable && isImageMessage && autoplayAnimatedImages) {
+            resolvedMode = ImageContentRenderer.Mode.ANIMATED_THUMBNAIL
```

Comment:
this feels like something the factory could/should be setting https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt#L525 (as it's not _bind time_ dependent)